src/components/Forms/Onboarding/index.js

```diff
@@ -98,6 +98,11 @@ export default class Form extends React.Component<
     this.setState({ showAdvanced: true });
   };
 
+  handleFinish = () => {
+    localStorage.setItem('chainName', this.state.name);
+    //history.push("/"); // not working here :(
```

Comment:
If the idea is to redirect the user to the `/` route, then use RR4’s [Redirect](https://reacttraining.com/react-router/web/api/Redirect) component. In general, let’s avoid mutating the `history` object because we might see some side-effects.